pyproject.toml

```diff
@@ -88,39 +88,57 @@ Documentation = "https://papis.readthedocs.io"
 Repository = "https://github.com/papis/papis"
 
 [project.optional-dependencies]
-develop = [
+# Dev environment -- install if you're working on papis
+develop = ["papis[lint,typing,test]"]
+# Style enforcement
+lint = [
     "flake8",
     "flake8-bugbear",
     "flake8-quotes",
     "Flake8-pyproject",
-    "mypy>=0.7",
     "pep8-naming",
     "pylint",
+]
+# Test suite
+test = [
     "pytest",
     "pytest-cov",
+]
+# Type-checking
+typing = [
+    "mypy>=0.7",
     "types-beautifulsoup4",
     "types-Pygments",
     "types-docutils",
-    "types-python-slugify",
```

Comment:
Was `types-docutils` necessary in the end?

Reply:
yes, it was